unit/util/irep.cpp

```diff
@@ -22,7 +22,7 @@ SCENARIO("irept_memory", "[core][utils][irept]")
       const std::size_t ref_count_size = 0;
 #endif
 
-#ifndef USE_STRING
+#ifndef USE_STD_STRING
```

Comment:
Side note: I’d really love if we could just get rid of `USE_STD_STRING` already. I’m not entirely convinced we keep it around for anything other than nostalgia at this point.

Reply:
I believe the only use case for it is / was debugging problems in the `dtringt` implementation.  I am not sure how many issues it will find that have not already been found.